crates/common/src/lib.rs

```diff
@@ -374,9 +374,9 @@ impl std::ops::SubAssign<u64> for BlockNumber {
     }
 }
 
-impl From<EthereumBlockNumber> for ethers::types::BlockId {
+impl From<EthereumBlockNumber> for BlockId {
```

Comment:
It's likely we can delete the `EthereumXXX` types now? Let me check the rest of the PR..